config/locales/en/main.yml

```diff
@@ -647,6 +647,7 @@ en:
         qtype_name: "The kind of information the user will be entering."
         access_level: "Inherit: Same access level as parent form. Private: No one can access the data via the API, regardless of parent form access level."
         metadata_type: "A special value that can be pre-filled into this question. If chosen, this question will be automatically hidden and not required, and any conditions will be removed."
+        reference: "A url that supports the question, must start with http or https."
```

Comment:
Slight edits: `A URL that supports the question. Must start with http:// or https://.`